doc/scoring.rst

```diff
@@ -61,48 +61,52 @@ look like this::
+calculated and printed under ``Mean CV scores``. In the example above, there is
+only one score metric; 'auc'. If more than one score metric was defined in
+``problem.py`` (see :ref:`score types <score-types>`), scores for all the score
+metrics will be printed.
 
 ``Bagged scores`` are calculated by combining the predictions of the 8 folds
 and using the combined prediction to calculate the score. For regression
-problems the combined prediction is just the mean of the predictions and
+problems the combined prediction is the mean of the predictions and
 for classification problems, it is the mean probability of each class. For
-detection problems the combined prediction calculation is more complex. See
+detection problems, the combined prediction calculation is more complex. See
 the `source code 
 <https://github.com/paris-saclay-cds/ramp-workflow/blob/12512a3192bcc515c2da956a6a6704849cdadeee/rampwf/prediction_types/detection.py#L37>`_
 for more details.
 
-For example, the Titanic challenge aims to predict whether or no each
-passenger survived. For each fold, different survival predictions are made for
-the test data. This is because each model is different as it was trained using
-different data). The probality of each classification (survived or did not
-survive), from the 8 predictions, is averaged and the classification computed
-with the new average probabilities. This is done for each sample in the test
-data and the new 'combined prediction' is used to calculate the 'bagged' score.
-This differs slightly for the validation score because the validation datasets
-will be different between each fold, and samples may or may not overlap between
-folds. In cases where there is only one prediction for a validation sample, the
-combined prediction will simply be the single prediction.
+For example, the Titanic challenge aims to predict whether or not each
+passenger survived. For each CV fold, different survival predictions are made
+for the test data. This is because for each CV fold, the model is different as
+it was trained using different data. The probality of each classification
+(survived or did not survive), computed from the 8 CV models, is averaged for
+every sample in the test dataset. The classification label is
+then computed using the new averaged probabilities. This new 'combined
+prediction' is used to calculate the 'bagged' score. The validation bagged
+score is calculated similarly, though there is a slight variation because the
+validation datasets differ between each CV fold. Validation
+samples may or may not overlap between CV folds. In cases where a validation
+sample was present in only one CV fold, there is only one prediction for that
+sample. The combined prediction for this sample will simply be that single
+prediction.
 
 Note that technically this is not what bagging means, but the name is used for
 historical reasons.
 
-RAMP event submissions
-======================
+RAMP Studio submissions
```

Comment:
ramp.studio is one instance of the RAMP app. I would not suggest it's **the** only one.